lib/ansible/utils/encrypt.py

```diff
@@ -162,3 +165,22 @@ def keyczar_decrypt(key, msg):
         return key.Decrypt(msg)
     except key_errors.InvalidSignatureError:
         raise AnsibleError("decryption failed")
+
+
+DEFAULT_PASSWORD_LENGTH = 20
+
+
+def random_password(length=DEFAULT_PASSWORD_LENGTH, chars=C.DEFAULT_PASSWORD_CHARS):
+    '''Return a random password string of length containing only chars
+
+    :kwarg length: The number of characters in the new password.  Defaults to 20.
+    :kwarg chars: The characters to choose from.  The default is all ascii
+        letters, ascii digits, and these symbols ``.,:-_``
+
+    .. note: this was moved from the old ansible utils code, as nothing
+        else appeared to use it.
```

Comment:
Since there was something else to change still, go ahead and get rid of this ```.. note:: [...]``` as well.  We've moved it back into ansible.utils, now ;-)